microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -1424,16 +1424,28 @@
+                      <strong>{{ _('How do we calculate your diversity value?') }}</strong>
                   </p>
                   <p>
-                      {{ _('The number of microbes found in your sample can be very close or vastly different from this average, and it doesn\'t necessarily mean something is right or wrong with you. We are all different, and this number is affected by many different factors.') }}
+                      {{ _('Diversity can be measured in a lot of different ways. We measure it by first taking the DNA sequences from your sample and figuring out which Bacteria and Archaea they may have come from. This produces a good estimate of how many of each kind of microbe you have. The measure we use, called Shannon\'s index, takes into account how many different kinds of microbes make up your gut community (also known as richness) and how relatively abundant each of those microbes are (also known as evenness).') }}
+                  </p>
+                  <p>
+                      {{ _('This number isn\'t exact for many reasons. However, because it\'s calculated in the same way for every sample in our dataset, it provides insight into how your microbial diversity compares to others\'. For example, the <span id="average-us-tooltip" class="orange-text" data-bs-toggle="tooltip" data-bs-html="true" data-bs-placement="top" title="Based on fecal samples from Microsetta participants in the United States">average</span> diversity value for people living in the US in our dataset is <strong id="average-american-observed-otus">FILLIN</strong>, which is lower than we find in people living a more hunter-gatherer lifestyle like the <span id="taxa-tooltip" class="orange-text" data-bs-toggle="tooltip" data-bs-html="true" data-bs-placement="top" title="The Hadza are an indigenous hunter-gather group in Tanzania. The summary of the Hadza microbiome is based on Smits et al. Science 2017">Hadza</span> people of Tanzania, who have an average value of <strong id="average-hadza-observed-otus">FILLIN</strong>. ') }}
+                  </p>
+                  <p>
+                      <strong>{{ _('What does this number mean for my health?') }}</strong>
+                  </p>
+                  <p>
+                      {{ _('This number reflects the variety of Bacteria and Archaea present in your gut at the time of testing. Currently, there\'s no single established standard for how gut microbiome diversity relates to health. While some studies suggest a potential link between higher diversity and specific health benefits, more research is needed to understand this connection.') }}
```

Comment:
```suggestion
                      {{ _('This number reflects the variety of Bacteria and Archaea observed in your fecal specimen at the time of testing. Currently, there\'s no standard for how gut microbiome diversity relates to health. While some studies suggest a potential link between higher diversity and specific health benefits, that is not true universally -- more research is needed to understand the relationship. For example, it may be that the specific microbes present are more important than having many different types of microbes, much like how in a garden you care about the exact plants growing not that there is a random assortment of plants.') }}
```